src/compiler/tsbuild.ts

```diff
@@ -16,6 +16,7 @@ namespace ts {
          */
         OutOfDateWithPrepend,
         OutputMissing,
+        ErrorReadingFile,

```

Comment:
Nit: I might call this "ErrorReadingBuildInfo"

Reply:
Made this generic for future